### PR TITLE
Postgres unit test generator for collect step

### PR DESCRIPTION
**Pull request by qvad**

### CODE REVIEW

File: src/actions/collects/pg_unit.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+            # cbo query?
+            if self.config.enable_statistics:
+                result_file.write(self.add_semicolon(ENABLE_STATISTICS_HINT))
+                result_file.write("\n")
+
+            result_file.write("\n")
+
+            previous_tag = None
+            for query in queries:
```
Comment (gauravk-in) [inline src/actions/collects/pg_unit.py:66]: For each query, we can add a FIXME comment if default plan is not the same as best plan. We can add the expected best plan in the comment.

File: src/actions/collects/pg_unit.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            result_file.write(f"SET statement_timeout = '{self.config.ddl_query_timeout}s'\n\n")
+            for model_query in create_queries:
+                if model_query.startswith("--"):
+                    result_file.write(model_query)
+                else:
+                    result_file.write(self.add_semicolon(' '.join(model_query.split())))
+                result_file.write("\n")
+
+            result_file.write("\n")
+            result_file.write("-- TBD: ADD STATISTICS IMPORT QUERIES\n")
```
Comment (gauravk-in) [inline src/actions/collects/pg_unit.py:48]: Updated the cbo_stat_load tool to generate sql statements for importing statistics.

File: src/actions/collects/pg_unit.py (hunk 1 above)
Comment (gauravk-in) [inline src/actions/collects/pg_unit.py:66]: I am thinking if it is a good idea to add the query id in a comment too. One problem I had was in finding the matching query in the TAQO result.

File: src/actions/collect.py [hunk 3]
```diff
@@ -187,10 +205,10 @@ def evaluate_optimizations(self, connection, cur, original_query):
 
             # check that execution plan is unique
             evaluate_sql(cur, optimization.get_heuristic_explain())
-            execution_plan = database.get_execution_plan(
+            optimization.basic_execution_plan = database.get_execution_plan(
```
Comment (gauravk-in) [inline src/actions/collect.py:208]: We should use EXPLAIN (costs off) so tests are more stable. Small cost changes that do not flip the query plan should not make the tests fail.

File: src/actions/collects/pg_unit.py (hunk 2 above)
Comment (qvad) [reply src/actions/collects/pg_unit.py:48]: Statistics will be added in separate script by replacing this line

**Pull request merged**